test/uk/gov/hmrc/helptosavefrontend/connectors/HelpToSaveConnectorSpec.scala

```diff
@@ -111,7 +111,7 @@ class HelpToSaveConnectorSpec extends TestSupport with GeneratorDrivenPropertyCh
           forAll(ineligibleResponseGen){ response ⇒
             val reason = Ineligible(response)
 
-            mockHttpGet(eligibilityURL)(Some(HttpResponse(200, responseJson = Some(Json.toJson(response)))))
+            mockHttpGet(eligibilityURL)(Some(HttpResponse(200, responseJson = Some(Json.toJson(ECResponseHolder(Some(response)))))))
```

Comment:
Seem to be missing a unit test for the case when `None` is returned by the backend

Reply:
Done